<launch>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find rags_ros)/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find rags_ros)/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find rags_ros)/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find rags_ros)/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find rags_ros)/base_local_planner_params.yaml" command="load" />
    
    <remap from="map" to="/map" />
  	<!-- START: Use with RAGS -->
<!--	  <remap from="cmd_vel" to="cmd_vel_raw"/>-->
    <!-- END: Use with RAGS -->
  </node>
</launch>
